draft-ietf-acme-acme.md

```diff
@@ -1981,8 +1981,8 @@ Content-Type: application/jose+json
 
 Note that because the identifier in a pre-authorization request is
 the exact identifier to be included in the authorization object,
-pre-authorization cannot be used to authorize issuance with wildcard
-DNS identifiers.
+pre-authorization cannot be used to authorize issuance of certificates
+containing wildcard domain names.
```

Comment:
This is a very minor nit, but is the term "wildcard domain name" common?  I thought that you could either have a "wildcard DNS identifier" in a certificate, or simply a "wildcard certificate" but that domain names were never wildcard.

Reply:
For better or for worse, "wildcard domain name" is the term that wound up codified in the BRs:

> Wildcard Domain Name: A Domain Name consisting of a single asterisk character followed by a single full stop character (“*.”) followed by a Fully-Qualified Domain Name.

Since this is the most rigorous definition of a wildcard available, we're starting to standardize around it, e.g. also in 6844bis. It has the downside that a "wildcard domain name" is not actually a "domain name," but it's progress anyhow.

Reply:
By the way, this is just harmonizing the language in different sections. In {{order-objects}} we talk about "wildcard domain names" and define them, but in {{pre-authorization}} we were talking about "wildcard DNS identifiers," which aren't defined anywhere but mean the same thing.

Reply:
Cool